pages/maker/maker-layout.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
```

Comment:
One thing here is that we never want to have anything other than "page-level" components in the `/pages` directory. This is a next.js thing, where each JS file in `/pages` is meant to be an actual "route". Doing this could actually have unintended consequences on things like bundle sizes and tree splitting. So, all of this code should be in `@containers/maker` or `@components/maker`.

Reply:
Agree, will merge this file into the components folder.